### DISCUSSION TITLE
Auto run is not working after update

### DISCUSSION THREAD

**Discussion by pnath29** [zhongyang219/TrafficMonitor#943] [Q&A]
![Untitled](https://user-images.githubusercontent.com/96072755/145850628-8742dddc-3f84-4fd4-b852-29104a49ffc3.png)

![image](https://user-images.githubusercontent.com/96072755/145850901-750c6fa7-968f-4cd1-9523-1207325dd9c2.png)

**Auto run is turning off automatically after the system reboot.**

**Comment by pnath29** [comment]
Someone please help

**Reply by david086t** [reply]
+1

**Reply by IMXEren** [reply]
https://youtu.be/0i2U2A6UsDE

**Reply by IMXEren** [reply]
Hope this will help you
